chromium/popup.html

```diff
@@ -19,13 +19,13 @@
     <h1 data-i18n="about_ext_name"></h1>
 </header>
 
-<section id="disableButton" class="options">
+<section id="disableButton" class="options" style="display:none;">
```

Comment:
No need of a semicolon here.

Reply:
I think either way is correct. Would you mind motivating why you would like the semi-colon removed?